.github/PR_TEMPLATE/pr-template.md

```diff
@@ -0,0 +1,15 @@
+- **What kind of change does this PR introduce:** 
```

Comment:
It will be better if we come up with checkboxes inside templates like this one.
https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/

Reply:
Sure Just give me sec

Reply:
Nice